lib/jsonapi/utils/dash.ex

```diff
@@ -0,0 +1,29 @@
+defmodule JSONAPI.Utils.Dash do
```

Comment:
Does this do more than the existing https://github.com/jeregrine/jsonapi/blob/d4371362ecaadaece27f4d640722f0beae90e6ad/lib/jsonapi/utils/underscore.ex#L1 ?

Reply:
:ping_pong: @jeregrine

Reply:
Lets merge this into one module, if you feel better renaming it thats fine just make sure the tests pass. But since they both use the same configuration and both have functionally the same responsibility I think having one will be better.